pydoctor/templatewriter/writer.py

```diff
@@ -108,7 +109,16 @@ def writeSummaryPages(self, system: model.System) -> None:
                 # not using missing_ok=True because that was only added in Python 3.8 and we still support Python 3.6
             except FileNotFoundError:
                 pass
-            root_module_path.symlink_to('index.html')
+
+            # When support for Python 3.9 and older is dropped use
+            # pathlib.Path.hardlink_to() instead.
+            x = list(root_module_path.parent.glob('*'))
+            print(f'TTTT\n{root_module_path=} {x=}')
+            print(f'{root_module_path.exists()=}')
+            os.link(
```

Comment:
OK, I need your help here. First of all I think I confounded the arguments in `os.link()`. So I changed them but this does not solve the problem.

The value of `root_module_path` is `PosixPath('apidocs/basic.html')` which does not exist. That seems to be the problem. But I do not understand why it does not exist or even if it have to exists.
Of course because of that the `os.link()` can not work.

Reply:
Make totally sense. Thanks for analyzing. Because this is not your own code, are you sure that your solution will do the job? Or I might should do some more research if there is a better location in code to call writeIndividualFiles()?